EightMinuteEmpireLegends/Map/Map.cpp

```diff
@@ -0,0 +1,260 @@
+#include "Map.h"
```

Comment:
Perhaps it's better not to push this and only push the mapLoader files, as when jforty pushes it might result in conflict. If what he pushes doesn't match what you wrote then we can fix later. What do you think?

Reply:
It's only a logical connection, it is actually referencing jforty code. It is because i am referencing the code from another project.

Reply:
I must have made a mistake pushing, let me double check.